.travis_scripts/integration_test.sh

```diff
@@ -5,8 +5,8 @@ set -e
 sudo /etc/init.d/mysql stop
 git clone https://github.com/streamr-dev/streamr-docker-dev.git
 sudo ifconfig docker0 10.200.10.1/24
-"$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh" start 5
-"$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh" start ganache
+"$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh" start --all
```

Comment:
@hpihkala I notice this starts both `ethereum-watcher` and `ethereum-watcher-local`, is this correct?

Reply:
I can’t imagine both are necessary. I remember seeing some question about it on Slack. Probably the ”local” is the right one. Don’t know who added them both and why. Anyway it’s some old garbage in the docker env, unrelated to any recent change.